docs/claim-transparency.md

```diff
@@ -0,0 +1,345 @@
+the transparent release project our goal is to provide transparency into the build and release
+processes. In our solution, in each step of the process software artifacts are being reviewed and
+audited and the result, a claim about the security and privacy properties of the software artifact,
+is signed and published into a [transparency log](https://continusec.com/static/VerifiableDataStructures.pdf).
+The goal is to make these claims easily discoverable so that anyone can use the claims in the
+assessment of privacy policies.
+
+## The Claim Format
+
+We propose the following structure, based on the [in-toto Statement standard](https://github.com/in-toto/attestation/blob/main/spec/README.md#statement),
+for specifying security and privacy claims. This format is meant to be generic and allow specifying
+different types of claims.
+
+```json
+{
+  "_type": "https://in-toto.io/Statement/v0.1",
+  "subject": [{ ... }],
+
+  "predicateType": "https://github.com/project-oak/transparent-release/schema/claim/v1",
+  "predicate": {
+    "issuer": {
+      "id": "<URI>"
+    },
+    "claimType": "<URI>",
+    "metadata": {
+      "issuedOn": "<TIMESTAMP>",
+      "validFrom": "<TIMESTAMP>",
+      "expiresOn": "<TIMESTAMP>",
+    },
+    "claimSpec": { /* object */ },
+    "evidence": [
+      {
+        "role": "<STRING>",
+        "uri": "<URI>",
+        "digest": { /* DigestSet */ }
+      }
+    ]
+  }
+}
+```
+
+### Fields
+
+This section describes the semantics of each field in the claim format:
+
+- **subject** _(array of objects, required)_:
+  Set of artifacts that the claim applies to. Usually only one artifact (e.g., source code, or some binary).
+  - **subject[*].digest** and **subject[*].name** as defined by Statement in the in-toto standard.
+- **issuer** _(object, required)_:
+  Identifies the entity (person, organization, or an automated tool) that reviewed/audited/examined
+  the artifact in the subject of this claim.
+  - **issuer.id** _(string ([TypeURI](https://github.com/in-toto/attestation/blob/main/spec/field_types.md#TypeURI)), required)_:
+    URI indicating the issuer’s identity (e.g., mailto:issuer@example.com).
+- **claimType** _(string ([TypeURI](https://github.com/in-toto/attestation/blob/main/spec/field_types.md#TypeURI)), required)_:
+  URI indicating what type of claim was issued. It determines the meaning of claimSpec and evidence below.
+- **metadata** _(object, required)_:
+  Additional details about the claim.
+
+  - **metadata.issuedOn** _(string ([Timestamp](https://github.com/in-toto/attestation/blob/main/spec/field_types.md#Timestamp)), required)_:
+    The timestamp of when the assessment of the artifact was performed. In most cases, the
+    assessment involves manually auditing or reviewing some source code. The timestamp in this case
+    does not have to be the accurate moment the review process started, but only the date on which
+    the review was done.
```

Comment:
Not sure what it means that the timestamp can be the date, do you mean it should be truncated to the beginning of a day? That seems complicated because of timezones. Perhaps simpler to just have this the time at which the claim is actually signed, which is a definite instant in time.

Reply:
Then the question will be who signs it, and when should it be signed, etc. Feel free to create a PR, once this one is merged, and fix the sentence if you think this is not accurate :)